cfgov/paying_for_college/jinja2/paying-for-college/choose-a-student-loan.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'layout-full.html' %}
+{% import 'templates/render_block.html' as render_block with context %}
+
+{% block css -%}
+    {{ super() }}
+    <link rel="stylesheet" href="{{ static('apps/paying-for-college/css/main.css') }}">
+{%- endblock css %}
+
+{% block content_modifiers -%}
+    {{ super() }} content__hide-horizontal-overflow
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom 
+{%- endblock %}
+
+{% block content_main -%}
+    {% for block in page.header -%}
+        {% if 'featured_content' in block.block_type %}
+            <div class="block
+                        block__border
+                        {{ 'u-mt0' if loop.first }}">
+                        {# u-mt0 is used instead of block__flush-top to keep top border #}
+                {% include_block block %}
+            </div>
+        {% else %}
+            {{ render_block.render(block, loop.index) }}
+        {% endif %}
+    {%- endfor %}
+
+    <div class="wrapper content_wrapper" id="choose-a-student-loan">
+
+    {% for block in page.content %}
+        {{ render_block.render(block, loop.index) }}
+    {%- endfor %}
+
+        <section class="legal-disclaimer">
+            <h5>Legal Disclaimer:</h5>
+            <p>The content on this page provides general consumer information. It is not legal advice or regulatory guidance. The CFPB updates this information periodically. This information may include links or references to third-party resources or content. We do not endorse the third-party or guarantee the accuracy of this third-party information. There may be other resources that also serve your needs.</p>
+        </section>
+    </div>
+
+{% endblock %}
+
+{% block javascript scoped %}
+    {{ super() }}
+    <script async>
+      if ( document.body.parentElement.className.indexOf( 'no-js' ) === -1 ) {
+        !function(){
+          {# Include site-wide JavaScript. #}
+          var s = [
+            '{{ static("apps/paying-for-college/js/quiz.js") }}'
```

Comment:
Why is this JavaScript file being included here in the template, instead of defined as part of the `page_js` [here](https://github.com/cfpb/cfgov-refresh/blob/c8841386ab6afb47b9f5f12e8db995576c780478/cfgov/paying_for_college/models/pages.py#L94), like secondary-navigation.js? A similar comment applies to the other templates which include JS like this. It would be better if we standardize on one method for including Page-specific JS files.

Reply:
Agreed.

Reply:
@chosak @Scotchester 
Assets added to the `page_js` page value (which gets [rolled into](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L315-L316) the page's `media` value) [must be available](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_layouts/base.html#L305-L309) at the `/static/js/routes/on-demand/` asset path. The on-demand namespace appears to be reserved for sitewide js modules, such as header.js, footer.js and exapandable.js, so it doesn't seem to be the right spot for page-specific js.

The template method we used to grab assets from the /apps/ namespace was copied from [other spots](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/jinja2/regulations3k/search-regulations.html#L216-L232) where app-specific js was needed. 

Is there a better or more standard way to deliver the app-namespaced js?